docs/AccelNNPAHowToUseAndTest.md

```diff
@@ -22,22 +22,26 @@ cmake --build . --target check-onnx-lit
 
 ### Numerical tests
 
-Numerical tests for NNPA are provided in `test/accelerators/NNPA/numerical`. Currently tests for MatMul2D, Gemm, LSTM, and GRU are provided and run by using following command. These tests check if a zDNN instruction is generated in shared library in adition to accurary check.
+Numerical tests for NNPA are provided in `test/accelerators/NNPA/numerical`. Currently tests for Conv2D, MatMul2D, Gemm, LSTM, and GRU are provided and run by using following command. These tests check if a zDNN instruction is generated in shared library in adition to accurary check.
 
 ```
 cmake --build . --config Release --target check-onnx-numerical-nnpa
 ```
 
 These tests uses the same test code with numerical tests for CPU (`test/modellib` and `test/numerial`), but uses different cmake file(`test/accelerator/NNPA/numerical/CMakeLists.txt`).
 
+##### Conv2D
```

Comment:
Side note, have you though of adding an `-maccel=NNPA` option that then would automatically set the current specialized options?

Reply:
Do you mean using only `-maccel=NNPA` without using the environment variable? Yes. it should be possible.

Reply:
@imaihal 
I leave it up to you. The solution you have is clearly working, and the cmake does add the flags. It is just if we have to test one manually, we have to turn on the right flags. This is why I though that we could just use the `-maccel=NNPA` to automatically set all the options. Again, either way works, whatever you like is good.

Reply:
@AlexandreEichenberger  I agree with you. I updated code to set them automatically when setting `--maccel=NNPA`. I kept original environment variables since they may be useful in the future.